tests/modules/suite2p_registration/test_suite2p_registration_suite2p_utils.py

```diff
@@ -131,6 +132,19 @@ def shift_frame_mock(frame, dy, dx):
         self.assertEqual(output.shape[1], self.n_frames)
         self.assertTrue(np.all(np.equal(output, np.load(self.mock_data_loc))))
 
+    def test_trim_frames(self):
+        """Test that empty frames are properly removed from the data.
+        """
+        # Create a bunch of frames with random values that have a "high value".
+        test_data = 100 * self.rng.random(size=(self.n_frames,
+                                                self.xy_shape,
+                                                self.xy_shape))
+        # Create a well separated frame.
+        test_data[10, :, :] *= 0.1
+        frames = trim_frames(test_data)
+        # Test that our different frame is removed.
+        self.assertEqual(len(test_data) - 1, len(frames))
```

Comment:
Do you mind additionally testing that it removed the frame we expected it to remove (just being paranoid)

Reply:
Changed up the test a bit. Take a look.